<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>portSettingsDialog</class>
 <widget class="QDialog" name="portSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial port settings</string>
  </property>
  <widget class="QGroupBox" name="selectSerialPortBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Serial port</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="serialPortInfoListBox"/>
    </item>
    <item>
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="manufacturerLabel">
      <property name="text">
       <string>Manufacturer:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="serialNumberLabel">
      <property name="text">
       <string>Serial Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="locationLabel">
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="vidLabel">
      <property name="text">
       <string>Vendor ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pidLabel">
      <property name="text">
       <string>Product ID:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameters</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="baudRateLabel">
       <property name="text">
        <string>BaudRate:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dataBitsLabel">
       <property name="text">
        <string>Data Bits:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="parityLabel">
       <property name="text">
        <string>Parity:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="stopBitsLabel">
       <property name="text">
        <string>Stop bits:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="flowControlLabel">
       <property name="text">
        <string>Flow control:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="flowControlBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="stopBitsBox"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="baudRateBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="dataBitsBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="parityBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="ApplyButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
